addons/stock_account/migrations/11.0.1.1/pre-migration.py

```diff
@@ -9,6 +9,15 @@ def update_fifo_cost_method(env):
     """The real costing method doesn't exist anymore, so the most similar one
     is 'average'. Values mapped according this.
     """
+    openupgrade.logged_query(
+        env.cr, """
+        UPDATE ir_default
+        SET json_value = '"fifo"'
+        WHERE field_id in (SELECT id FROM ir_model_fields 
+                         WHERE model = 'product.template' 
+                             AND name = 'cost_method')
```

Comment:
You are not checking the current default. What if it is set to something else but `"real"`?

Travis noticed trailing whitepace in your query.

Reply:
Right thanks